bin/plugin/commands/since.js

```diff
@@ -17,6 +18,10 @@ const { log, formats } = require( '../lib/logger' );
  */
 
 exports.options = [
+	{
+		argname: '-p, --plugin <plugin>',
+		description: 'Plugin name',
```

Comment:
This is not the plugin name, but plugin slug.
```suggestion
		description: 'Plugin slug',
```